policies/before_resolution/registry.rego

```diff
@@ -0,0 +1,37 @@
+package otel
+
+# A registry `attribute_group` containing at least one `ref` attribute is
```

Comment:
not blocking: would it be possible to link some docs on the policy syntax and the properties available on group/attribute/etc?

Reply:
@lmolkova, you will find some documentation I wrote on this topic here https://github.com/open-telemetry/weaver/tree/main/crates/weaver_checker 

The definition of the attributes for the violation entity is defined here https://github.com/open-telemetry/weaver/blob/main/crates/weaver_checker/src/violation.rs

In general, the policy file is based on the Rego language and its documentation is available here https://www.openpolicyagent.org/docs/latest/policy-language/